dvc/repo/get.py

```diff
@@ -49,12 +65,28 @@ def get(url, path, out=None, rev=None):
             # the same cache file might be used a few times in a directory.
             repo.cache.local.cache_types = ["reflink", "hardlink", "copy"]
 
-            o = repo.find_out_by_relpath(path)
+            output = None
+            output_error = None
+
+            try:
+                output = repo.find_out_by_relpath(path)
+            except OutputNotFoundError as ex:
```

Comment:
Do we need https://github.com/iterative/dvc/pull/2837/files#diff-d84a7ca40e278c5be8904f9aeb922da0R93 any more?

Reply:
No, we can raise it with path here https://github.com/iterative/dvc/pull/2837/files#diff-d84a7ca40e278c5be8904f9aeb922da0R83

Good idea